Bottom: 662a846c8120f9bd9b0ff20f430fa9ca3781869e
Top:    1801eac9cf7ae938e37f2283fd9e428216376a76
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 15:18:44 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index a3444c4..57da8e6 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -51,7 +51,7 @@ static const char *rpcb_netnametbl[] = {
 	"rpcbind", "portmapper", "sunrpc", NULL,
 };
 const rpcprog_t rpcb_prog = RPCBPROG;
-const rpcvers_t rpcb_version = RPCBVERS_4;
+const rpcvers_t rpcb_version = RPCBVERS;
 #else
 static const char *rpcb_pgmtbl[] = {
 	NULL,
